processor/cmd/root.go

```diff
@@ -76,6 +85,22 @@ func Process(_ *cobra.Command, _ []string) {
 	}
 	fmt.Println()
 
+	// Verify the legal hold data
+	if legalHoldSecret != "" {
```

Comment:
We may want an option to just check the HMACs without doing all the conversion to HTML.  That can be deferred to next version.

Reply:
Yeah I wanted to add a different command but also avoid adding to many changes.